src/main/kotlin/no/nav/familie/dokument/storage/StorageController.kt

```diff
@@ -21,56 +22,49 @@ import java.util.*
 @ProtectedWithClaims(issuer = "selvbetjening", claimMap = ["acr=Level4"])
 class StorageController(@Autowired val storage: AttachmentStorage,
                         @Autowired val contextHolder: TokenValidationContextHolder,
-                        @Value("\${attachment.max.size.mb}") val maxFileSizeInMb: Int)  {
+                        @Value("\${attachment.max.size.mb}") val maxFileSizeInMb: Int) {
 
     private val secureLogger = LoggerFactory.getLogger("secureLogger")
 
     /// TODO: "bucket"-path brukes ikke ennå. "familievedlegg" brukes alltid
     @PostMapping(path = ["{bucket}"],
                  consumes = [MediaType.MULTIPART_FORM_DATA_VALUE],
                  produces = [MediaType.APPLICATION_JSON_VALUE])
-    fun addAttachment(@PathVariable("bucket")bucket: String,
+    fun addAttachment(@PathVariable("bucket") bucket: String,
                       @RequestParam("file") multipartFile: MultipartFile): ResponseEntity<Map<String, String>> {
 
         if (multipartFile.isEmpty) {
-            log.info("Dokumentet som lastes opp er tomt - size: [{}] ", multipartFile.size)
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build()
+            throw IllegalArgumentException("Dokumentet som lastes opp er tomt - size: [${multipartFile.size}] ")
         }
 
         val bytes = multipartFile.bytes
-        val maxFileSizeInBytes = maxFileSizeInMb*1024*1024
+        val maxFileSizeInBytes = maxFileSizeInMb * 1024 * 1024
         log.debug("Dokument lastet opp med størrelse (bytes): " + bytes.size)
 
         if (bytes.size > maxFileSizeInBytes) {
-            throw IllegalArgumentException(HttpStatus.PAYLOAD_TOO_LARGE.toString())
+            throw IllegalArgumentException("Dokumentstørrelsen(${bytes.size} bytes) overstiger grensen(${maxFileSizeInMb} mb)")
```

Comment:
This will be logged in the secureLogger if we would use my ExceptionHandler. 
If so, I think it would be better if we log it and return BAD_REQUEST? @charliemidtlyng ?